package.json

```diff
@@ -33,6 +33,7 @@
     "linefeed"
   ],
   "devDependencies": {
-    "mocha": "*"
+    "ava": "^0.14.0",
```

Comment:
Isn't there a danger in leaving `*` versions on a project that hasn't been built in 2 years? Especially for dependencies that are still pre `1.0.0`?

Isn't there value in documenting which version was used the last time the tests passed?